app/keepers/keepers.go

```diff
@@ -361,13 +361,14 @@ func NewAppKeeper(
 		appCodec, appKeepers.keys[routertypes.StoreKey],
 		appKeepers.GetSubspace(routertypes.ModuleName),
 		appKeepers.TransferKeeper,
-		// appKeepers.IBCKeeper.ChannelKeeper,
+		appKeepers.IBCKeeper.ChannelKeeper,
 		appKeepers.DistrKeeper,
 	)
 
-	appKeepers.RouterModule = router.NewAppModule(appKeepers.RouterKeeper, transferIBCModule) // 0,
-	// routerkeeper.DefaultForwardTransferPacketTimeoutTimestamp,
-	// routerkeeper.DefaultRefundTransferPacketTimeoutTimestamp,
+	appKeepers.RouterModule = router.NewAppModule(appKeepers.RouterKeeper, transferIBCModule, 0,
```

Comment:
0 is for `retriesOnTimeout`, it should not be 0 ?

Reply:
i guess they put in the work to make it possible to retry we should at least use the feature

Reply:
```suggestion
	appKeepers.RouterModule = router.NewAppModule(appKeepers.RouterKeeper, transferIBCModule, 1,
```